sources/execDynawoAlgorithms.sh

```diff
@@ -77,12 +77,43 @@ setEnv() {
   export LD_LIBRARY_PATH=$DYNAWO_ALGORITHMS_INSTALL_DIR/lib:$LD_LIBRARY_PATH
 }
 
+find_and_call_timeline() {
+  if [ ! -d "$1" ]; then
+    return 1
+  fi
+  find $1 -name $2 | while read line; do
+    echo "Processing file '$line'"
+    python $DYNAWO_ALGORITHMS_INSTALL_DIR/bin/timelineFilter.py --timelineFile $line
```

Comment:
Maybe you should use $DYNAWO_ALGORITHMS_PYTHON_COMMAND

Reply:
It is not defined in the execution environment